File: src/test/java/com/weeth/domain/user/test/fixture/UserCardinalTestFixture.java
package com.weeth.domain.user.test.fixture;

import com.weeth.domain.user.domain.entity.Cardinal;
import com.weeth.domain.user.domain.entity.User;
import com.weeth.domain.user.domain.entity.UserCardinal;

public class UserCardinalTestFixture {

	public static UserCardinal linkUserCardinal(User user, Cardinal cardinal) {
		return new UserCardinal(user, cardinal);
	}
}

File: src/test/kotlin/com/weeth/domain/user/fixture/UserCardinalTestFixture.kt
package com.weeth.domain.user.fixture

import com.weeth.domain.user.domain.entity.Cardinal
import com.weeth.domain.user.domain.entity.User
import com.weeth.domain.user.domain.entity.UserCardinal

object UserCardinalTestFixture {
    fun linkUserCardinal(
        user: User,
        cardinal: Cardinal,
    ): UserCardinal = UserCardinal(user, cardinal)
}
